tensorflow/docs_src/get_started/get_started.md

```diff
@@ -29,7 +29,9 @@ working internally when you use the more compact higher level API.
 
 # Tensors
 
-The central unit of data in TensorFlow is the **tensor**. A tensor consists of a
+The central unit of data in TensorFlow is the **tensor**. The formal physicist's and 
+mathematician's definition of a tensor is a multilinear map; the tensors here are in fact the same 
```

Comment:
You can add a link to the Wikipedia page for tensor or something like that for clarity.